test/electricity/angle_of_light_deflection_in_prism_test.py

```diff
@@ -1,18 +1,19 @@
 from collections import namedtuple
 from pytest import approx, fixture, raises
+from sympy import pi
 from symplyphysics import (units, SI, convert_to, Quantity, errors, dimensionless,)
 from symplyphysics.laws.electricity import angle_of_light_deflection_in_prism as angle_law
 
 # Description
-## Consider a prism with an angle between the faces equal to 45 degree and a refractive index of 15.
-## The angle of deviation will be 630 degrees.
+## Consider a prism with an angle between the faces equal to 45 degree (pi / 4 radian) and a refractive index of 2.5.
```

Comment:
What about refractive index of 15? The doesn't seem to have physical sense. Could you explain it?

Reply:
> What about refractive index of 15? The doesn't seem to have physical sense. Could you explain it?
With a refractive index of 15, the angle is 630, which actually corresponds to an angle of 270 degrees (630 mod 360). To avoid such large values of the deflection angle, I reduced the refractive index.

Reply:
> What about refractive index of 15? The doesn't seem to have physical sense. Could you explain it?

With a refractive index of 15, the angle is 630, which actually corresponds to an angle of 270 degrees (630 mod 360). To avoid such large values of the deflection angle, I reduced the refractive index.

Reply:
I think you are wrong and refracted angles over 90 degrees result in full reflection of the ray.

Reply:
> I think you are wrong and refracted angles over 90 degrees result in full reflection of the ray.

Ok. I don't understand what the point of the remark is. What needs to be explained, shown, or calculated in the current test?

Reply:
Add short explanation to the law description.